### PR TITLE
PaymentForwarder: Implementing checksumming of customerId

### PR DESCRIPTION
**Pull request by villesundell**
per issue #43.

### CODE REVIEW

**Comment by villesundell** [comment]
Also, will implement checksumming for the Crowdsale, but in a separate branch.

File: ico/tests/contracts/test_forwarder.py [hunk 1]
```diff
@@ -105,7 +110,7 @@ def test_halt(web3, payment_forwarder, team_multisig, customer):
     team_multisig_begin = web3.eth.getBalance(team_multisig)
     payment_forwarder.transact({"from": team_multisig}).halt()
     with pytest.raises(TransactionFailed):
-        payment_forwarder.transact({"value": value, "from": customer}).pay(customer_id, customer)
+        payment_forwarder.transact({"value": value, "from": customer}).payWithoutChecksum(customer_id, customer)
 
 
 def test_unhalt(web3, payment_forwarder, team_multisig, customer):
```
Comment (miohtama) [inline ico/tests/contracts/test_forwarder.py]: Can you also add a reverse test that we check if Invested event comes through with a correct checksum and we can verify this in Python land?

Comment (villesundell) [reply ico/tests/contracts/test_forwarder.py]: Is this request satisfied by the recent commit?

**Comment by miohtama** [comment]
Make a flag that toggle this behavior on and off - default on

**Pull request merged**